modules/packages/MPI.chpl

```diff
@@ -29,7 +29,91 @@ routines ::
       MPI_Keyval_create
       MPI_Keyval_free
 
-since all of these are built around user-defined handlers, that we support.
+since all of these are built around user-defined handlers, that we do not support.
+
+We also include a few functions that are in later MPI standards. These are ::
+
+      MPI_Init_thread (and related constants)
+      MPI_Ibarrier
+
+Compiler wrappers
+-----------------
+
+Using MPI requires pointing the compiler to the location of the
+MPI headers, and including the appropriate MPI libraries. This is
+often done by using compiler wrappers like "mpicc". Setting
+``CHPL_TARGET_COMPILER=mpi-gnu`` and ``CHPL_TARGET_ARCH=none`` will
+get Chapel to use these compilers.
```

Comment:
will get Chapel to use these compilers -> will get Chapel to use mpicc and assume that it works as gcc does